brainframe/cli/commands/backup.py

```diff
@@ -31,7 +31,7 @@ def backup():
 
     docker_compose.assert_installed(install_path)
 
-    dependencies.rsync.ensure(args.noninteractive, args.install_rsync)
+    dependencies.rsync.ensure(args.noninteractive, False)
```

Comment:
Is this set to False because without `rsync` the backup command simply won't work?

Reply:
This bool decides whether or not the CLI will attempt to install rsync for the user. I set this to false because we only supported installing rsync on Ubuntu 18.04 and 20.04, and those distributions can now use the deb instead.

Now that I think about it though, this is probably premature. People on 18.04 and 20.04 are still going to be using the Pip installation of the CLI until we can get a package repository set up, among other things. I'll change this back for the time being.